src/Plant/struct/struct_tree.jl

```diff
@@ -0,0 +1,14 @@
+# struct for a C3 tree
+Base.@kwdef mutable struct StructTree{FT<:AbstractFloat}
+    # tree information
+    age::Int = 10          # year | age
+    ba ::FT  = FT( 0.1)    # m²   | basal area
+    ga ::FT  = FT(50.0)    # m²   | ground area
+    h  ::FT  = FT( 8.0)    # m    | tree height
+
+    # tree formation from root to leaves
+    roots ::StructTreeRoot   = StructTreeRoot{FT,5}()           # | root struct with 5 layers by default
```

Comment:
```suggestion
    roots ::Root   = Root{FT,5}()           # | root struct with 5 layers by default
```
Id probably suggest a shorter name here, since it’s a property of `Tree`, the `Tree` in `TreeRoot` is inferred.